### DISCUSSION TITLE
Change port 8080 to something else

### DISCUSSION THREAD

**Discussion by freedomwarrior** [akvorado/akvorado#416] [Q&A]
Hello.
Please, can you tell me, how can I change default port 8080 to something else and what changes I need to do?
Thank you.

**Comment by freedomwarrior** [comment]
I try to change port 8080 to port 8082 in docker-compose.yml but then when I run docker-compose up I got this errors in logs:
`Error: unable to fetch configuration file: Get "http://akvorado-orchestrator:8082/api/v0/orchestrator/configuration/inlet": dial tcp 172.18.0.8:8082: connect: connection refused`
`akvorado_akvorado-inlet_1 exited with code 1`
`akvorado_akvorado-orchestrator_1 exited with code 1`

**Reply by vincentbernat** [reply]
You need to change only the last section:

```yaml
    ports:
      # Port 8080 is considered private as it exposes sensible unauthenticated
      # services (ClickHouse, configuration, ...). Therefore, it is only exposed
      # on the loopback. Port 8081 is the one you can expose to users. If you
      # want to change these ports, leave "8080/tcp" and "8081/tcp" unchanged:
      # you can modify the IP/port on the left.
      - 127.0.0.1:8080:8080/tcp
      - 8081:8081/tcp
```

**Reply by freedomwarrior** [reply]
It works, thank you!
